transit_odp/publish/tests/test_seasonal_service_views.py

```diff
@@ -147,6 +148,39 @@ def test_delete_last_seasonal_service(self, publish_client):
             == self.NUMBER_SEASONAL_SERVICE_PER_ORGANISATION - 1
         )
 
+    def test_transition_page_when_delete_seasonal_service(self, publish_client):
+        SEASONAL_SERVICE_UPGRADE = 6
```

Comment:
I thought we needed 11 seasonal services to make two pages?

Reply:
There are already 15 seasonal services + 6 => 21 seasonal services 3 pages.